src/types.ts

```diff
@@ -368,7 +368,18 @@ export interface KAPLAYCtx<
      *
      * @param a The angle to rotate by. Defaults to 0.
```

Comment:
might be worthwhile to note that angles for this are in degrees, some other game libs use radians

Reply:
true

Reply:
doesn't it already siay (in degrees)

Reply:
oh duh, it was the line that got cut off by the GitHub diff